File: Day26/Solution.java
public class Solution {

    public static void main(String[] args) {
        /* Enter your code here. Read input from STDIN. Print output to STDOUT. Your class should be named Solution. */
        Scanner sc = new Scanner(System.in);
        //da,ma,ya are for the actual date
        //de, me, ye are for expected book return date
        int da,ma,ya,de,me,ye;
        da = sc.nextInt();
        ma = sc.nextInt();
        ya = sc.nextInt();
        de = sc.nextInt();
        me = sc.nextInt();
        ye = sc.nextInt();

        int fine = 0;

        if(ye==ya){

            if(me < ma){
                fine = (ma - me) * 500;
            }
            else if( (me == ma) && (de < da) ){
                fine = (da - de) * 15;
            }

        }
        else if(ye < ya){
            fine = 10000;
        }
        System.out.println(fine);
    }
}

File: Day26/Solution.py
da, ma, ya = input().split(' ')
da = int(da)
ma = int(ma)
ya = int(ya)
de, me, ye = input().split(' ')
de = int(de)
me = int(me)
ye = int(ye)
fine = 0
if(ye==ya):
    if(me < ma):
        fine = (ma - me) * 500
    elif((me == ma) and (de < da)):
        fine = (da - de) * 15
elif(ye < ya):
    fine = 10000

print( fine )
